### PR TITLE
feat: migration step1 metamask (ui)

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary

https://app.clickup.com/t/39xfapr

Notice that we need to add new env variable on vercel if we want to change the `TRANSACTION_FEE`

- [x] Implements the Ui required for the task
- [x] Tests
- [x] move the fee to an env variable
- [x] The select address modal still needs a different title 

## Checklist

-   [ ] My changes look good in both light AND dark mode
-   [ ] The change is not hardcoded to a single network, but has multi-asset in mind
-   [ ] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: src/domains/migration/components/MigrationConnectStep/MigrationConnectStep.tsx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+	<button type="button" className="group relative overflow-hidden rounded-lg">
+		<span className="pointer-events-none absolute left-0 top-0 block h-full w-full bg-gradient-to-r from-[#FFDB80] to-[#F27C0B] opacity-100 transition-all duration-200 ease-in-out group-hover:opacity-0" />
+		<span className="pointer-events-none absolute left-0 top-0 block h-full w-full bg-gradient-to-r from-[#FFD486] to-[#4381C0] opacity-0 transition-all duration-200 ease-in-out group-hover:opacity-100" />
+		<div className="flex space-x-2 py-[14px] px-5">
+			<span className="relative">
+				<MetamaskLogo />
+			</span>
+			<span className="relative font-semibold text-white">{children}</span>
+		</div>
+	</button>
```
Comment (dated) [inline src/domains/migration/components/MigrationConnectStep/MigrationConnectStep.tsx:31]:
```suggestion
	<button type="button" className="group relative overflow-hidden rounded-lg bg-gradient-to-r from-[#FFDB80] to-theme-warning-800">
		<span className="absolute inset-0 bg-gradient-to-r from-theme-warning-200 to-theme-navy-500 opacity-0 transition-all duration-200 ease-in-out group-hover:opacity-100" />

		<div className="relative flex space-x-2 py-3.5 px-5">
			<MetamaskLogo />
			<span className="font-semibold text-white">{children}</span>
		</div>
	</button>
```

File: src/domains/migration/i18n.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+			FORM: {
+				AMOUNT_YOU_GET: "Amount You Get",
+				AMOUNT_YOU_SEND: "Amount You Send",
+				ARK_ADDRESS: "ARK Address",
+				METAMASK: {
+					CONNECT_WALLET: "Connect Wallet",
+					INSTALL_METAMASK: "Install MetaMask",
+					MESSAGES: {
+						NEEDS_METAMASK:
+							"In order to complete the Migration, you must first install MetaMask. Learn more in our <linkMigrationGuide>Migration Guide.</linkMigrationGuide>",
```
Comment (dated) [inline src/domains/migration/i18n.ts:23]:
```suggestion
							"In order to complete the Migration, you must first install MetaMask. Learn more in our <linkMigrationGuide>Migration Guide</linkMigrationGuide>.",
```

File: src/domains/migration/components/MigrationConnectStep/MigrationConnectStep.tsx [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+							<div className="font-semibold text-theme-secondary-900 dark:text-theme-secondary-200">
+								<Amount ticker="ARK" value={walletBalance} />
+							</div>
+						</div>
+						<div className="space-y-2 text-right">
+							<div className="text-sm font-semibold text-theme-secondary-500 dark:text-theme-secondary-700">
+								{t("TRANSACTION.TRANSACTION_FEE")}
+							</div>
+							<div className="font-semibold text-theme-secondary-900 dark:text-theme-secondary-200">
+								<Amount ticker="ARK" value={-TRANSACTION_FEE} showSign />
```
Comment (dated) [inline src/domains/migration/components/MigrationConnectStep/MigrationConnectStep.tsx:148]:
```suggestion
								<Amount ticker="ARK" value={TRANSACTION_FEE * -1} showSign isNegative />
```

**Review by samharperpittam — CHANGES_REQUESTED** [review]
The `Polygon Migration Address` field should be a different colour when blurred making the text easier to read. It's more noticeable in dark mode but also an issue in light mode.

Design
image

Imp
image

**Review by samharperpittam — CHANGES_REQUESTED** [review]
On mobile (375) we should remove the border around the content which will give us more space. See the images below

Des
image


Imp
image

Also on mobile the text should read `In order to complete the Migration, you must first install the MetaMask mobile app or use a Web3-compatible browser. Learn more in our Migration Guide.`

**Review by samharperpittam — APPROVED** [review]

**Review by samharperpittam — APPROVED** [review]

**Review by dated — APPROVED** [review]

**Pull request merged**